docs/visual-basic/getting-started/whats-new.md

```diff
@@ -53,6 +56,21 @@ Bit-shift operators, loop variable declaration
 Visual Basic / Visual Studio .NET 2002
 The first release of Visual Basic .NET
 
+## Visual Basic 16
+
```

Comment:
```suggestion
Visual Basic 16.0 focuses on supplying more of the features of the Visual Basic Runtime (microsoft.visualbasic.dll) to .NET Core and is the first version of Visual Basic focused on .NET Core. Many portions of the Visual Basic Runtime depend on WinForms and these will be added in a later version of Visual Basic. 

**Comments allowed in more places within statements**
```

@paul1956 Do you have a better header here?

Reply:
With some minor changes many WinForms applications with work, a reasonable set of instructions with necessary changes could be linked to. Also what was listed as 15.8 is in VB16 and should not be lost. The main thing missing is the My namespace and MsgBox listing it acknowledges what issues people might face.